qa/rpc-tests/spentindex.py

```diff
@@ -131,7 +131,7 @@ def run_test(self):
         assert_equal(len(coinbase_tx['outputs']), 2)
         assert_equal(coinbase_tx['outputs'][0]['index'], 0)
         assert_equal(coinbase_tx['outputs'][1]['index'], 1)
-        assert_equal(coinbase_tx['outputs'][1]['satoshis'], 2.5*COIN)
+        assert_equal(coinbase_tx['outputs'][1]['zatoshis'], 2.5*COIN)
```

Comment:
Revert this and other renames involving outputs of RPC calls (including everything in this file).